{
  Type: ArgumentException,
  Message: The Message,
  ParamName: The parameter,
  StackTrace:
at Tests.MethodThatThrowsArgumentException()
at InnerVerifier.Throws(Action target)
}